src/main/java/gyro/google/compute/InstanceGroupResource.java

```diff
@@ -238,6 +294,55 @@ public void copyFrom(InstanceGroup instanceGroup) {
         }
     }
 
+    private void addInstances(List<String> instances) throws Exception {
+        Compute client = createComputeClient();
+        waitForCompletion(
+            client,
+            client.instanceGroups()
+                .addInstances(getProjectId(), getZone(), getName(), new InstanceGroupsAddInstancesRequest()
+                    .setInstances(instances.stream()
+                        .map(instance -> new InstanceReference().setInstance(instance))
+                        .collect(Collectors.toList())
+                    )
+                ).execute()
+        );
+    }
+
+    private void removeInstances(List<String> instances) throws Exception {
+        Compute client = createComputeClient();
```

Comment:
The client could be a parameter, enabling it to be reused, as this method in called from `update` which creates its own clients.

Reply:
Done.